README.md

```diff
@@ -143,6 +143,34 @@ The web application becomes available on the IP addresses of the Docker containe
 
 As a convenience, `sandbox run` and `sandbox debug` reports each of the above mappings along with IP addresses.
 
+## Roles
+
+ConductR uses roles to determine which bundle can be loaded and scaled on which node. For more information about refer to [Managing services documentation](http://conductr.typesafe.com/docs/1.0.x/ManageServices#Roles).
+
+By default each ConductR node is allowed to load and scale bundles with the role `web`. The bundle role can be specified with the [sbt-bundle](https://github.com/sbt/sbt-bundle) key `BundleKeys.roles`. When deploying bundles with a different role than `web` then this role needs to added to the ConductR configuration by specifying the `SandboxKeys.conductrRoles` key:
+
+```scala
+SandboxKeys.conductrRoles in Global := Seq(Set("muchMem", "myDB"))
```

Comment:
Can we not collect the roles from the modules of the project and auto-populate this setting - as we do for `ports`?